devtools/conda-envs/test_env.yaml

```diff
@@ -27,9 +27,11 @@ dependencies:
     # JSON caching of residue templates
   - tinydb
 
-    # openff toolkit and force fields
-  - openff-toolkit >=0.7.1
+    # OpenFF toolkit and force fields
+  - openff-toolkit >=0.10.3|<=0.10.0
   - openff-forcefields >=1.2.0
+  - pip:
+    - git+https://github.com/openforcefield/openff-toolkit.git@topology-biopolymer-refactor
```

Comment:
If you don't want to shoulder the complexity of testing against an unreleased version, you can grab it with a similar command locally (`pip install git+https://github.com/openforcefield/openff-toolkit.git@topology-biopolymer-refactor`) and have a look around for yourself.

Reply:
And for the record: I'm not suggesting this change be merged in; I want to show in the context of this PR that it should pass with the new API. I will revert this before merging if approved.

Reply:
Thanks for this! We should test against both the current and preview versions, ideally. But to do that, we can't just eliminate testing the current version---I'll have to rework the test matrix.